domain/robstoll-lib/atrium-domain-robstoll-lib-common/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/throwableAssertions.kt

```diff
@@ -0,0 +1,26 @@
+package ch.tutteli.atrium.domain.robstoll.lib.creating
+
+import ch.tutteli.atrium.api.fluent.en_GB.ExperimentalWithOptions
+import ch.tutteli.atrium.api.fluent.en_GB.withRepresentation
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+import ch.tutteli.atrium.domain.creating.changers.ChangedSubjectPostStep
+import ch.tutteli.atrium.domain.robstoll.lib.creating.throwable.thrown.creators.ThrowableThrownFailureHandler
+import ch.tutteli.atrium.reporting.RawString
+import ch.tutteli.atrium.translations.DescriptionThrowableAssertion
+import kotlin.reflect.KClass
+
+@UseExperimental(ExperimentalWithOptions::class)
+inline fun <TExpected : Throwable> _cause(
+    expect: Expect<Throwable>,
+    expectedType: KClass<TExpected>
+): ChangedSubjectPostStep<*, TExpected> =
+    ExpectImpl.feature.manualFeature(expect, DescriptionThrowableAssertion.OCCURRED_EXCEPTION_CAUSE) { cause }
+        .getExpectOfFeature()
+        .withRepresentation { it ?: RawString.create(DescriptionThrowableAssertion.NO_EXCEPTION_OCCURRED) }
```

Comment:
I actually like your attempt to be more descriptive than null, i.e. instead of 
```
expected that subject: () -> kotlin.Nothing        (readme.examples.ReadmeSpec$1$4$1 <1234789>)
◆ ▶ thrown exception when called: java.lang.IllegalArgumentException
    ◆ ▶ cause: null
         ◾ is instance of type: IllegalStateException (java.lang.IllegalStateException)
```
you report currently
```
expected that subject: () -> kotlin.Nothing        (readme.examples.ReadmeSpec$1$4$1 <1234789>)
◆ ▶ thrown exception when called: java.lang.IllegalArgumentException
    ◆ ▶ cause: no exception occurred
         ◾ is instance of type: IllegalStateException (java.lang.IllegalStateException)
```

I think we can further improve it by being a bit more related to the cause. How about changing it to 
```
    ◆ ▶ cause: ❗❗ not caused by another exception
```
If you agree (does not need to be in this PR), then add another entry to DescriptionThrowableAssertion